packages/tokens/bin/lib/formats/formatTs.ts

```diff
@@ -0,0 +1,43 @@
+import {camelCase} from 'lodash';
+
+import {isTokenEnum, isTokenGroup, Token, TokenEnum, TokenGroup, TokenList} from './token';
+
+export const valueIncludesUnit = (value: string | number) => /px$|%$/.test(value.toString());
+export const valueIsVariable = (value: string | number) => /^var\(--(.+)\)$/.test(value.toString());
```

Comment:
I find them used only in this file you can delete the export
```suggestion
const valueIncludesUnit = (value: string | number) => /px$|%$/.test(value.toString());
const valueIsVariable = (value: string | number) => /^var\(--(.+)\)$/.test(value.toString());
```